packages/cypress-commands/src/setups/enableNetworkShim/createStateFromFixtures.js

```diff
@@ -0,0 +1,45 @@
+import { isStubMode, getNetworkFixturesDir } from './utils'
+
+function parseFixtureFiles(fileNames) {
+    return cy
+        .all(
+            ...fileNames.map(fileName => () =>
+                cy.readFile(`${getNetworkFixturesDir()}/requests/${fileName}`)
```

Comment:
I would change this to `cy.fixture`. On one side I had to read the docs first to know that this function does more than its name reveals, it's also parsing JSON. On the other side, as these files are in the fixtures folder, I think `.fixture` is more appropriate